docs/sphinx/about/whats-new.rst

```diff
@@ -41,12 +41,12 @@ Bug fixes and improvements:
 
 Documentation improvements:
 
-* updated website and URL links for new ``openmicroscopy.org`` website
-* added missing Andor SIF to supported formats page
-* added a new page ``Working with whole slide images`` outlining the API support 
+* updated website and URL links for new <http://www.openmicroscopy.org>`_ website
```

Comment:
Maybe `OME Website <https://www.openmicroscopy.org>`_ as the brackets are currently rendered.